libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NativeSkuTokenProvider.kt

```diff
@@ -0,0 +1,7 @@
+package com.mapbox.services.android.navigation.v5.navigation
+
+import com.mapbox.navigator.SkuTokenSource
+
+internal class NativeSkuTokenProvider : SkuTokenSource() {
+    override fun getToken(): String = ""
+}
```

Comment:
Are we adding this new file to `Legacy` SDK? 🤔

Reply:
Yeah so everything keeps working in the legacy. In any case, we don't support SKU in the legacy so this should do the trick 👀 https://github.com/mapbox/mapbox-navigation-android/pull/2955#discussion_r424601044